Rakefile

```diff
@@ -0,0 +1,12 @@
+desc "Build docs and install to Xcode"
```

Comment:
I guess you use `rake` cuz they did?
I guess they think it's portable (it can only ever run on OSX after all)?
I would have gone with `/bin/sh`, but it really does not matter.